.golangci.yml

```diff
@@ -14,6 +14,9 @@ issues:
     - appendAssign
     - halp  # Make misspell be quiet about this.
     - exitAfterDefer # Potentially useful but not in any cases it fires right now.
+    - unused-parameter # Quite a few of these, some legit, many are fulfilling interfaces
```

Comment:
We _might_ want to have a think about this; it can be a useful check, but it fires on a lot of cases that are fulfilling interfaces etc and changing `func write(force bool) error` to `func write(_ bool) error` is a step backwards in readability.

Reply:
Tbh, I come across these in IJ and it’s cleverer about realising that it’s due to an interface. So personally happy to disable